### ISSUE TITLE
olStyleUtils test seems to be flaky

### ISSUE THREAD

**Issue by taulinger** [ldbv-by/bav4#2185]
```
absoluteolStyleUtils.test.3071735055.js markerStyleFunction should return a style with a specific anchor FAILED
	Expected null to equal [ 32, 32 ].
	<Jasmine>
	./test/modules/olMap/util/olStyleUtils.test.js/</<@/tmp/_karma_webpack_146093/commons.js:466534:33
Chrome Headless 120.0.6099.28 (Linux x86_64): Executed 3480 of 3480 SUCCESS (1.311 secs / 2 mins 28.312 secs)
Firefox 119.0 (Linux x86_64): Executed 3159 of 3480 (1 FAILED) (0 secs / 1 min 5.266 secs)
Firefox 119.0 (Linux x86_64) absoluteolStyleUtils.test.3071735055.js markerStyleFunction should return a style with a specific anchor FAILED
	Expected null to equal [ 32, 32 ].

```

**Comment by taulinger** [comment]
Error still occurs:

```
Firefox 124.0 (Linux x86_64) absoluteolStyleUtils.test.3071735055.js markerStyleFunction should return a style with a default anchor FAILED
	Expected null to equal [ 16, 16 ].
	<Jasmine>
	./test/modules/olMap/util/olStyleUtils.test.js/</<@/tmp/_karma_webpack_849631/commons.js:481819:33
===

**Comment by taulinger** [comment]
@thiloSchlemmer Fails periodically with FF124 now (tagged this issue as `urgent`)

**Comment by taulinger** [comment]
fixed with #2322
